README.md

```diff
@@ -112,16 +113,84 @@ Write a conditional statement that...
 8.  Write your javascript solution below
 ```js
 // your answer here
+let num = 50;
+if (num % 3 === 0 && num % 5 === 0){
+    console.log('FizzBuzz');
+}
+else if (num % 3 === 0){
+    console.log('Fizz');
+}
+else if (num % 5 === 0){
+    console.log('Buzz');
+}
+else {
+    console.log(num);
+}
 ```
 
 #### BONUS
 9.  Research a [loop](https://javascript.info/while-for) so that your condition runs on every number from 0 to 100
 ```js
 // your answer here
+//for loop
+for (let num = 1; num <= 100; num++){
+    if (num % 3 === 0 && num % 5 === 0){
+        console.log('FizzBuzz');
+    }
+    else if (num % 3 === 0){
+        console.log('Fizz');
+    }  
+    else if (num % 5 === 0){
+        console.log('Buzz');
+    }
+    else {
+        console.log(num);
+    }
+}
+
+//while loop
+let num = 1;
+while (num <= 100){
+    if (num % 3 === 0 && num % 5 === 0){
+        console.log('FizzBuzz');
+    }
+    else if (num % 3 === 0){
+        console.log('Fizz');
+    }  
+    else if (num % 5 === 0){
+        console.log('Buzz');
+    }
+    else {
+        console.log(num);
+    }
+
+    num++;
+}
 ```
 10.  Research a [function](https://javascript.info/function-basics) so that your condition runs on every number from 0 to whatever number is passed into the function
 ```js
 // your answer here
+//function
+function checkConditions(num){
```

Comment:
this function doesn't run every number from 0 to the entered number. It is done by using your code from question 9, and replacing the 100 with the input value.